test/IntelFPGAMemoryAttributes.ll

```diff
@@ -1,3 +1,180 @@
+; Source
```

Comment:
I'd suggest adding a link to the compiler repository & attaching the compilation command. If suitable, the same applies to `IntelFPGAMemoryAttributesForStruct.ll`

Reply:
Done